### PR TITLE
[tests]: Improve test reliability on OS X / BSD

### LINKED ISSUE #174: Make tests pass also on Mac OS
According Travis history we are experiencing continuous builds failure since 10 months. It is not really the case since I was able to run the testsuite on my laptop (using Debian Stretch) without failing tests for CPython 2.7, 3.5 and PyPy.
Sometimes there are 1 or 2 tests failing so we should just identify those and make them reliable.

I would happy to have this issue as a release goal for circuits 3.3, if it seems feasible.

### PR DESCRIPTION
**Pull request by prologic**
Fixes #174

### CODE REVIEW

File: tests/core/test_bridge.py [hunk 1]
```diff
@@ -3,18 +3,21 @@
...
 
 from os import getpid
+from time import sleep
 
 
 from circuits import ipc, Component, Event
 
 
+if pytest.PLATFORM == "win32":
+    pytest.skip("Unsupported Platform")
```
Comment (apollo13) [inline tests/core/test_bridge.py]: this needs to be `pytestmark = pytest.mark.skip("Unsupported Platform")` on newer pytest versions.

File: tests/node/test_server.py [hunk 2]
```diff
@@ -15,6 +15,10 @@
 from circuits.node import Node
 
 
+if PLATFORM == 'win32':
+    skip('Broken on Windows')
```
Comment (apollo13) [inline tests/node/test_server.py]: Same as above.

Comment (prologic) [inline tests/node/test_server.py]: Okay

File: tests/core/test_bridge.py (hunk 1 above)
Comment (prologic) [inline tests/core/test_bridge.py]: Thanks!

**Pull request merged**